omc3/correction/sequence_evaluation.py

```diff
@@ -0,0 +1,289 @@
+"""
```

Comment:
Also new issue: to be checked if it can be improved/made faster by Laurents suggestion in https://github.com/MethodicalAcceleratorDesign/MAD-X/issues/832#issuecomment-662996548

Reply:
Let's make this into a new issue and not take care of it for this PR